src/include/duckdb/parallel/pipeline.hpp

```diff
@@ -68,6 +68,8 @@ class Pipeline {
 	bool IsFinished() {
 		return finished;
 	}
+	//! Returns query progress
+	int GetProgress(bool &supported);
```

Comment:
Reversing these two arguments (`bool GetProgress(int &progress)`) is more common/standardized I feel, perhaps better to switch them?